functions/index.js

```diff
@@ -1,9 +1,7 @@
 const functions = require("firebase-functions");
-const { helloWorld } = require("./controllers/helloworld/index");
-const { app } = require("./controllers/expressExample/index");
+const { app } = require("./controllers/endpoints/index");
 const { verifyRecaptcha } = require("./controllers/VerifyRecaptcha/index");
 
```

Comment:
This is separate from this story but it's a quick fix but can we add
`app.disable("x-powered-by");
app.use(helmet());`
to the verifyRecaptcha app similarly to how it was used in expressExample/endpoints.